graphify-out/GRAPH_REPORT.md

```
@@ -0,0 +1,314 @@
+# Graph Report - Collective-Modding-Toolkit  (2026-06-12)
+
+## Corpus Check
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)